src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const smallImages = document.querySelectorAll('.list-item__link');
+
+const imagesLinksArray = Array.from(smallImages);
```

Comment:
You can do it like that 
```suggestion
const imagesLinksArray = Array.from(smallImages, (item) => item.getAttribute('href'));
```